bindings/node.js/src/kzg.cxx

```diff
@@ -149,6 +149,10 @@ Napi::Value LoadTrustedSetup(const Napi::CallbackInfo& info) {
       return env.Undefined();
   }
   data->is_setup = true;
+  int close_err = fclose(file_handle);
+  if (close_err != 0) {
+      Napi::Error::New(env, "Error closing trusted setup file: " + file_path).ThrowAsJavaScriptException();
+  }
```

Comment:
```suggestion
  if (close_err != 0) {
      Napi::Error::New(env, "Error closing trusted setup file: " + file_path).ThrowAsJavaScriptException();
      return env.Undefined();
  }
```

Let's complete this error check correctly with a return value so that future changes to this function are not prone to bugs.

Reply:
Good idea.  I also moved the `is_setup` to just before the final return so it will "not be setup" when it returns early